sources/install/package_wordlists.sh

```diff
@@ -49,8 +49,7 @@ function install_seclists() {
     mkdir -p /usr/share/wordlists
     ln -v -s /opt/seclists /usr/share/seclists
     ln -v -s /opt/seclists /usr/share/wordlists/seclists
```

Comment:
these can be removed, I don't even know why they were created in the first place to be honest

Reply:
Yes, here i modify the following lines : 
47    cd /opt/seclists || exit
by 
    cd /opt/lists/seclists || exit

And 
50    add-test-command "[[ -f '/usr/share/wordlists/rockyou.txt' ]]"
51    add-test-command "[[ -d '/opt/seclists/Discovery/' ]]"
by
    add-test-command "[[ -f '/opt/lists/rockyou.txt' ]]"
    add-test-command "[[ -d '/opt/lists/seclists/Discovery/' ]]"